### DISCUSSION TITLE
Force in the bathing scenario

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#163] [Other Q&A] [answered]
Regarding the force in the bathing scenario, I noticed that in the official document “RCareWorld/template/test_bathing.py,” it states, "The threshold for a comfortable force on the human body is set to 1-6N."

At the same time, I saw "force percentage" information in the output log of the graphical interface while running the bathing program. I'm not quite sure about the relationship between these two. Does "force percentage" reach 100% when the force exceeds 6N?

In simple terms, as a competitor, should I just focus on keeping the output of `SpongeAttr.GetForce()` within the range of 1-6N?

**Comment by YoruCathy** [comment, marked as answer]
Force percentage means in all the time steps happened so far, the portion of the forces that are with in the threshold. You can use both of the numbers for reference.

**Comment by TomatoPicker278** [comment]
Thank you for your response; I understand what you mean. There is a passage on the official webpage that states:

"Perform the full-body bed bathing by sliding the scrubber over the entire body of the manikin (35 pts):
Body coverage: (n%)*10 (n = area covered with water/entire top body)
Force threshold: (m%)*25 (m = timestep of forces within the threshold/entire timesteps when the scrubber is in contact with the human)"

In my question, the "force percentage" refers specifically to the m% mentioned here.
